README.md

```diff
@@ -172,25 +172,32 @@ Below is a table of currently used environment variables for starting the servic
-| TEST_USER_ID                   | User ID for a test account used as a header for most publication tests.                    |
+| Variable                       | Description                                                                                                                            |
+|:-------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------|
+| CLIENT_ID                      | As above                                                                                                                               |
+| CLIENT_SECRET                  | As above                                                                                                                               |
+| APP_URI                        | As above                                                                                                                               |
+| SUBSCRIPTION_MANAGEMENT_AZ_API | As above                                                                                                                               |
+| TENANT_ID                      | As above                                                                                                                               |
+| ACCOUNT_MANAGEMENT_AZ_API      | As above                                                                                                                               |
+| PUBLICATION_SERVICES_AZ_API    | As above                                                                                                                               |
+| SYSTEM_ADMIN_PROVENANCE_ID     | Value for the provenance of a system admin used as a header on authentication-bound tests.                                             |
+| TEST_USER_ID                   | User ID for a test account used as a header for most publication tests.                                                                |
+| CLIENT_ID_FT                   | Client ID of external service used for authentication with data-management application in the functional tests.                        |
+| CLIENT_SECRET_FT               | Client secret of external service.used for authentication with data-management application in the functional tests.                    |
+| CONNECTION_STRING              | As above                                                                                                                               |
+| STORAGE_ACCOUNT_NAME           | As above. Only required when running the functional tests in the staging environment in pipeline.                                      |
+| MANAGED_IDENTITY_CLIENT_ID     | Azure Blob Storage managed identity client ID. Only required when running the functional tests in the staging environment in pipeline. |
```

Comment:
I think MANAGED_IDENTITY_CLIENT_ID can be removed here

Reply:
Removed